<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Triaxial Plot</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Logo-Ntnu.svg</normaloff>Logo-Ntnu.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QGridLayout" name="widgetGrid"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>ID</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Lable</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Depth [m]</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Comment</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Header Info</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Calibration Info</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Su [kPa]</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Strain rate [%]</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>$\delta{V} [cm^3]$</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>$\epsilon_{vol} [%]$</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>$\delta{e}/e_{0} [-]$</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
